src/server/web_server.js

```diff
@@ -149,17 +155,21 @@ app.use(express_compress());
 ////////////
 // ROUTES //
 ////////////
-if (prom_reports.instance().enabled()) {
+if (prom_reporting.is_enabled()) {
+    // Enable proxying for all metrics servers
+    app.use('/metrics/web_server', express_proxy(`localhost:${config.WS_METRICS_SERVER_PORT}`));
+    app.use('/metrics/bg_workers', express_proxy(`localhost:${config.BG_METRICS_SERVER_PORT}`));
+    app.use('/metrics/hosted_agents', express_proxy(`localhost:${config.HA_METRICS_SERVER_PORT}`));
+
+    // TODO: this is here for backward competability, remove when all is working and tested
     app.get('/metrics', function(req, res) {
-        res.set('Content-Type', prom_reports.instance().client().register.contentType);
-        res.end(prom_reports.instance().client().register.metrics());
+        res.set('Content-Type', 'text/plain');
```

Comment:
why ? what was the client (lib) default content-type ?

Reply:
text/plain